src/Compilers/CSharp/Portable/Compilation/MemberSemanticModel.cs

```diff
@@ -340,6 +340,10 @@ private static Binder GetEnclosingBinderInternalWithinRoot(SyntaxNode node, int
                 {
                     binder = rootBinder.GetBinder(current);
                 }
+                else if (current is CheckedExpressionSyntax)
```

Comment:
>current is CheckedExpressionSyntax [](http://example.com/codeflow?start=25&length=34)

We probably should check that we are inside parens. We do similar check for argument list and switch section, for example. if we are on the checked/unchecked keyword, we haven't entered the context yet, that should be possible to observe through a test.